### PR TITLE
[Cisco Umbrella] Add Cisco Umbrella package

### LINKED ISSUE #646: Migrate Cisco umbrella package
# Integration release checklist

This checklist is intended for integrations maintainers to ensure consistency
when creating or updating a Package, Module or Dataset for an Integration.

### Migration Checklist

- [ ] Change follows the [contributing guidelines](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md)
- [ ] Supported versions of the monitoring target are documented
- [ ] Generated output for at least 1 log file exists
- [ ] Documentation explains how to configure monitoring target to work with the integration
- [ ] Fields follow [ECS](https://github.com/elastic/ecs) and [naming conventions](https://www.elastic.co/guide/en/beats/devguide/master/event-conventions.html)
- [ ] At least a manual test with ES / Kibana / Agent has been performed.
- [ ] Required Kibana version set to:
- [ ] Screenshot of the "Add Integration" page on Fleet added to PR

### Dashboards

- [ ] Dashboards exists
- [ ] Screenshots added to manifest
- [ ] Data stream filters added to visualizations

### Data Streams
- umbrella
  - [ ] [Pipeline tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/pipeline_testing.md) added
  - Inputs
    - s3
      - [ ] [System tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/system_testing.md) added

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Add initial package for Cisco Umbrella.  Migrated from Filebeat Module

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Author's Checklist

- [ ]

## How to test this PR locally
```
cd integrations/packages/cisco_umbrella
elastic-package build && elastic-package stack down && elastic-package stack up --version 7.16.0-SNAPSHOT -d -v && eval "$(elastic-package stack shellinit)" && elastic-package test  -v
```

## Related issues

- Closes #646

### CODE REVIEW

File: packages/cisco_umbrella/_dev/build/build.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+dependencies:
+  ecs:
+    reference: git@1.11
```
Comment (legoguy1000) [inline packages/cisco_umbrella/_dev/build/build.yml:3]: Need to update to 1.12

File: packages/cisco_umbrella/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+description: Pipeline for Cisco Umbrella
+
+processors:
+# ECS event.ingested
+- set:
+    field: event.ingested
+    value: '{{_ingest.timestamp}}'
+- set:
+    field: ecs.version
+    value: '1.11.0'
```
Comment (legoguy1000) [inline packages/cisco_umbrella/data_stream/log/elasticsearch/ingest_pipeline/default.yml:11]: Need to update to 1.12

**Comment by andrewkroh** [comment]
run tests

**Review by P1llus — APPROVED** [review]

File: packages/cloudflare/changelog.yml [hunk 3]
```diff
@@ -3,4 +3,4 @@
   changes:
     - description: initial release
       type: enhancement # can be one of: enhancement, bugfix, breaking-change
-      link: https://github.com/elastic/integrations/pull/984
+      link: https://github.com/elastic/integrations/pull/1646
```
Comment (legoguy1000) [inline packages/cloudflare/changelog.yml:6]: This was my bad

Comment (P1llus) [reply packages/cloudflare/changelog.yml:6]: Let's fix that in a second PR, much appreciated :)

Comment (legoguy1000) [reply packages/cloudflare/changelog.yml:6]: https://github.com/elastic/integrations/pull/1720

**Pull request merged**